integration_tests/CMakeLists.txt

```
@@ -508,6 +508,8 @@ RUN(NAME expr_02u            LABELS cpython llvm llvm_jit c NOFAST)
 RUN(NAME expr_03u            LABELS cpython llvm llvm_jit c NOFAST)
 RUN(NAME expr_04u            LABELS cpython llvm llvm_jit c)
 
+RUN(NAME list_01             LABELS cpython llvm llvm_jit c)
```

It seems this test fails with`c` backend at the CI. So let's remove it.

```suggestion
RUN(NAME list_01             LABELS cpython llvm llvm_jit)
```